CONTRIBUTING.md

```diff
@@ -182,15 +182,15 @@ Using this you can test the entire package and verify that all parts of the pack
 
 The testing script includes:
 
-- linting using [ESlint][eslint]
+- linting using [ESLint][eslint]
```

Comment:
This is correct, but "SASS" (meaning the CSS preprocessor) is incorrect: "Sass" is correct. And there are more.